.github/workflows/test.yaml

```
@@ -30,3 +30,15 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  conda-java-tests:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@python-3.12
```

Use `branch-24.12` after you merge in the upstream and retarget this PR:
```suggestion
    uses: rapidsai/shared-workflows/.github/workflows/custom-job.yaml@branch-24.12
```